src/test/java/net/imagej/ops/topology/EulerCharacteristic26NFloatingTest.java

```diff
@@ -0,0 +1,127 @@
+package net.imagej.ops.topology;
+
+import net.imagej.ops.AbstractOpTest;
+import net.imglib2.RandomAccess;
+import net.imglib2.img.Img;
+import net.imglib2.img.array.ArrayImgs;
+import net.imglib2.type.logic.BitType;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Tests for the {@link EulerCharacteristic26NFloating EulerCharacteristic26NFloating} class
+ *
+ * @author Richard Domander
+ */
+public class EulerCharacteristic26NFloatingTest extends AbstractOpTest {
+    @Test(expected = IllegalArgumentException.class)
+    public void testConforms() throws AssertionError {
+        final Img<BitType> img = ArrayImgs.bits(3, 3);
+
+        ops.run(EulerCharacteristic26NFloating.class, img);
+    }
+
+    /**
+     * Test with a single voxel (=solid cube) that floats in the middle of a 3x3x3 space
+     * <p>
+     * Here χ = β_0 - β_1 + β_2 = 1 - 0 + 0 = 1.
+     * The formula χ = vertices - edges + faces for surfaces of polyhedra doesn't apply because the cuboid is solid.
```

Comment:
Always previous your javadoc in your IDE; as written, the sentence starting with "The formula" will _not_ begin on a new line.